libledger/DBInitializer.cpp

```diff
@@ -173,11 +173,9 @@ void DBInitializer::initSQLStorage()
     sqlStorage->setChannelRPCServer(m_channelRPCServer);
     sqlStorage->setTopic(m_param->mutableStorageParam().topic);
     sqlStorage->setFatalHandler([](std::exception& e) {
-        (void)e;
-        LOG(FATAL) << "Access amdb failed, exit";
-
-        // must use exit to avoid rasie failed
-        exit(1);
+        LOG(ERROR) << "Access amdb failed exit:" << e.what();

```

Comment:
why not use FATAL

Reply:
This error is possible, our code should handle this, make process exit normally with error log.